common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
@rabbah since the pluggability is another concern, do we want to mix these 2 concerns in a single PR ? Adding a pluggable solution now, would also mean to update the existing implementation based on logging to disk, making the PR a little too big IMHO.